metricbeat/module/coredns/stats/_meta/fields.yml

```diff
@@ -0,0 +1,71 @@
+- name: stats
+  type: group
+  description: >
+    Contains statistics related to the coreDNS service
+  release: beta
+  fields:
+    - name: panic.count.total
+      type: long
+      description: >
+         Total number of panics
+    - name: dns.request.count.total
+      type: long
+      description: >
+         Total query count
+    - name: dns.request.duration.sec.bucket
+      type: object
+      description: >
+         Request duration histogram buckets
+    - name: dns.request.duration.sec.sum
+      type: long
+      format: duration
+      description: >
+         Requests duration, sum of durations
+    - name: dns.request.duration.sec.count
+      type: long
+      description: >
+         Requests duration, number of requests
+    - name: dns.request.size.bytes
+      type: object
+      description: >
+         Request Size histogram buckets
+    - name: dns.request.do.count.total
+      type: long
+      description: >
+         Number of queries that have the DO bit set
+    - name: dns.request.type.count.total
+      type: long
+      description: >
+          Counter of queries per zone and type
+    - name: dns.request.type
+      type: keyword
+      description: >
+        Holds the query type of the request
+    - name: dns.response.rcode.count.total
+      type: long
+      description: >
+        Counter of responses per zone and rcode
+    - name: dns.response.rcode
+      type: keyword
+      description: >
+        Holds the rcode of the response
+    - name: dns.request.family
+      type: keyword
+      description: >
+        The address family of the transport (1 = IP (IP version 4), 2 = IP6 (IP version 6))
+    - name: dns.response.size.bytes
+      type: object
+      description: >
+          Response Size histogram buckets
```

Comment:
We should probably have a field mapping like this one for histograms:
```
    - name: dns.response.size.bytes.bucket.*
      type: object
      object_type: double
      description: >
          Response Size histogram buckets
    - name: dns.response.size.bytes.sum
      type: double
      description: >
          Response Size histogram sum
    - name: dns.response.size.bytes.count
      type: double
      description: >
          Response Size histogram count
```

Reply:
following this approach I see the system test to be failing on the check about fields documentation.
https://travis-ci.org/elastic/beats/jobs/498584317#L5377
Can we handle this somehow?

Reply:
It should work, but maybe the problem is in the dot in the `8.192`, that would match `dns.response.size.bytes.bucket.*.*`, but not `dns.response.size.bytes.bucket.*`.

We should take rid of these dots in the histogram fields first :slightly_smiling_face:

Reply:
:+1: